modules/psap-special-resource-operator-installing-using-cli.adoc

```diff
@@ -0,0 +1,123 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/psap-special-resource-operator.adoc
+
+[id="installing-the-special-resource-operator-using-cli_{context}"]
+= Installing the Special Resource Operator by using the CLI
+
+As a cluster administrator, you can install the Special Resource Operator (SRO) by using the {product-title} CLI.
+
+.Prerequisites
+
+* You have a running {product-title} cluster.
+* You installed the OpenShift CLI (`oc`).
+* You are logged into the OpenShift CLI as a user with `cluster-admin` privileges.
```

Comment:
@kmccarron-rh should it be {product-title} CLI (as on line 8) or OpenShift CLI, or simply `oc` CLI?

Reply:
OpenShift CLI per RH Style guide, so {product-title} wouldn't work here. 
There are many instances in our doc with the Prerequisite written as "You installed the OpenShift CLI (`oc`)."